includes/definitions/papouch-quido.yaml

```diff
@@ -0,0 +1,9 @@
+os: papouch-quido
+text: 'Papouch QUIDO'
+type: environment
+icon: papouch-tme
+mib_dir:
+    - papouch
+discovery:
+    - sysObjectID:
+        - .0.9.43.6.1.4.1
```

Comment:
This matches the TME devies too, you'll need to add additional match data like the tme detection